keyboards/handwired/hnah40/rules.mk

```
@@ -1,7 +1,15 @@
 # MCU name
 MCU = atmega328p
 
-BOOTLOADER = bootloadHID
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = USBasp
```

I think this should be switched back.

```suggestion
BOOTLOADER = bootloadHID
```

---

`PROGRAM_CMD` specifies USBasp, and the `:program` target is mentioned in the readme. @vuhopkep could you confirm which bootloader is present on this board?

---

Hi there, my board (hnah40) use usbasp from Plaid board